IVsTestingExtension/NuGet.Config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="globalPackagesFolder" value="gpf" />
```

Comment:
why?

Reply:
In my experience, this tool involves a lot of custom testing. 
Having the gpf isolated avoids getting too many xprivates inside the gpf. 

I'll remove it.